x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/get_is_alert_suppression_active.ts

```diff
@@ -13,7 +13,7 @@ import type { AlertSuppressionCamel } from '../../../../../common/api/detection_
 
 interface GetIsAlertSuppressionActiveParams {
   alertSuppression: AlertSuppressionCamel | undefined;
-  isFeatureDisabled: boolean | undefined;
```

Comment:
It doesn't look like we're using this field now (except in tests), but I assume we're keeping it around for the next round of suppression work?